terraform-plans/templates/github/gitignore.tftpl

```diff
@@ -0,0 +1,49 @@
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
+
+# Python Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Test files and directories
+.pytest_cache/
+.coverage
+.tox
+reports/
+**/report/
+htmlcov/
+.mypy_cache
+
+# python virtual environments (for local dev)
+.venv
+venv
+env
+
+# Build artefacts
+output/
+.build/
+build/
+*.charm
+*.snap
+# python build artefacts
+deb_dist/
+dist/
+*.egg-info/
+
+# Log files
+*.log
+
+# IDEs
+.idea/
+.vscode/
```

Comment:
nit:
I think this kind of gitignore it's responsibility of the developer to set a global ignore locally. There are too many possible IDEs to have an exhaustive list

Reply:
Yep I 100% agree :+1:  If there's no concern from others, I'm very happy to remove them.

Reply:
Updated :)